### ISSUE TITLE
Parsing URL from anrdoezrs.net

### ISSUE THREAD

**Issue by Germs2004** [uBlockOrigin/uBlock-issues#1096]
### Prerequisites

- [ ] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [ ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [ ] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [ ] uBlock Origin is the only extension
    - [ ] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [ ] I am running the latest version of uBlock Origin
- [ ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

On the Strict-Blocked page, uBlock attempts to parse out all the parameters.  If it finds a URL parameter, we can click the URL to go straight to the desired page, bypassing the ad site.

I found an ad site that has a URL integrated into its URL, not as a parameter.  It would be nice if that URL could be detected and put seperately on the uBlock page so I can click it.

### A specific URL where the issue occurs

Source URL:
https://www.indiegamebundles.com/fanatical-dollar-discovery-bundle-2/
(click the button that says "You can buy the Fanatical Dollar Discovery Bundle 2 here")

Blocked URL:
http://www.anrdoezrs.net/links/7179331/type/dlg/https://www.fanatical.com/en/bundle/dollar-discovery-2-bundle

URL that uBlock should parse out:
https://www.fanatical.com/en/bundle/dollar-discovery-2-bundle

### Expected behavior:

uBlock should let me click the link to "https://www.fanatical.com/en/bundle/dollar-discovery-2-bundle" from its block page

### Actual behavior:

uBlock doesn't parse out that URL because it isn't an HTTP parameter.

![image](https://user-images.githubusercontent.com/5519340/83799580-0647b000-a674-11ea-8a3c-1aeb1d2ca530.png)

### Your environment

* uBlock Origin version: 1.27.10
* Browser Name and version: Chrome 83.0.4103.61
* Operating System and version: Windows 10 1909

**Comment by uBlock-user** [comment]
> uBlock doesn't parse out that URL because it isn't an HTTP parameter.

No, because it's not a query string. Only query strings are parsed

**Comment by Germs2004** [comment]
Yes, I understand that only query strings are currently parsed.  That's why I created this ticket, to request that URLs also get parsed from the blocked URL string.  It's a useful feature for users to be able to either click "Temporarily Allow" or just quickly click the URL that they want to go to, bypassing the ad-site.
